File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// AdviceSlipApi2 SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class AdviceSlipApi2MakeContext
{
    public static function call(array $ctxmap, ?AdviceSlipApi2Context $basectx): AdviceSlipApi2Context
    {
        return new AdviceSlipApi2Context($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# AdviceSlipApi2 SDK utility: make_context
require_relative '../core/context'
module AdviceSlipApi2Utilities
  MakeContext = ->(ctxmap, basectx) {
    AdviceSlipApi2Context.new(ctxmap, basectx)
  }
end
